Mage.Sets/src/mage/cards/c/CrowdControlWarden.java

```diff
@@ -0,0 +1,116 @@
+package mage.cards.c;
+
+import java.util.UUID;
+
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.dynamicvalue.DynamicValue;
+import mage.abilities.dynamicvalue.common.PermanentsOnBattlefieldCount;
+import mage.abilities.effects.ReplacementEffectImpl;
+import mage.abilities.effects.common.counter.AddCountersSourceEffect;
+import mage.abilities.hint.Hint;
+import mage.abilities.hint.ValueHint;
+import mage.constants.*;
+import mage.abilities.costs.mana.ManaCostsImpl;
+import mage.abilities.keyword.DisguiseAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.counters.CounterType;
+import mage.filter.StaticFilters;
+import mage.game.Game;
+import mage.game.events.EntersTheBattlefieldEvent;
+import mage.game.events.GameEvent;
+import mage.game.permanent.Permanent;
+
+/**
+ * @author Cguy7777
+ */
+public final class CrowdControlWarden extends CardImpl {
+
+    private static final Hint hint = new ValueHint(
+            "Other creatures you control",
+            new PermanentsOnBattlefieldCount(StaticFilters.FILTER_CONTROLLED_ANOTHER_CREATURE));
+
+    public CrowdControlWarden(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{3}{G}{W}");
+
+        this.subtype.add(SubType.CENTAUR);
+        this.subtype.add(SubType.SOLDIER);
+        this.power = new MageInt(4);
+        this.toughness = new MageInt(4);
+
+        // As Crowd-Control Warden enters the battlefield or is turned face up, put X +1/+1 counters on it, where X is the number of other creatures you control.
+        Ability ability = new SimpleStaticAbility(Zone.ALL, new CrowdControlWardenReplacementEffect());
+        ability.setWorksFaceDown(true);
+        ability.addHint(hint);
```

Comment:
Make sure it doesn't show card hint in face down state (card must be hidden). If it does then report it as face down bug.

Reply:
Just double checked, the hint isn't shown for you or your opponent when face down.